wagtailmenus/models.py

```diff
@@ -106,6 +106,12 @@ class MenuItem(models.Model):
         blank=True,
         help_text=_("Must be set if you wish to link to a custom URL."),
     )
+    link_handle = models.SlugField(
+        max_length=100,
+        blank=True,
+        help_text=_(
```

Comment:
I would drop the 'Must be unique' bit from here, unless you intend to enforce some custom validation on clean(). But I don't see any reason why developers shouldn't be able to add unique values.